masonry/src/contexts.rs

```diff
@@ -859,6 +882,19 @@ impl LayoutCtx<'_> {
     }
 }
 
+impl ComposeCtx<'_> {
+    pub fn needs_compose(&self) -> bool {
+        self.widget_state.needs_compose
+    }
+
+    pub fn set_child_translation(&mut self, translation: Vec2) {
```

Comment:
```suggestion
    /// offsets the child by `translation` relative to the parents `window_origin`
    pub fn set_child_translation(&mut self, translation: Vec2) {
```

Reply:
(Don't want this to be a conflict point, but was this overlooked or do you want some different doc-comment?)

Reply:
Huh, that's odd, I thought I'd added a comment there.

... And, looking again, yes I did. The UI is just absolutely terrible at showing it.

Reply:
Ok, yeah indeed weird, I thought it would show `Outdated` or something, thanks :)